### PR TITLE
Autoformatting using uncrustify

### PR DESCRIPTION
**Pull request by ckelleyRH**
To run:
`yum install uncrustify`
`uncrustify --replace --no-backup -c - SSLVersionRange.h`
`Parsing: ./SSLVersionRange.h as language C-Header`

To validate:
`uncrustify --check -c - SSLVersionRange.h`
`Parsing: ./SSLVersionRange.h as language C-Header`
`PASS: ./SSLVersionRange.h (469 bytes)`

### CODE REVIEW

**Comment by ckelleyRH** [comment]
I used the default config for both Java and C, but it's highly configurable - [this is the config ](https://github.com/uncrustify/uncrustify/blob/master/forUncrustifySources.cfg)`uncrustify` use to uncrustify `uncrustify`.

It compiled at least :1st_place_medal:

**Comment by emaldona** [comment]
Regarding that default config for both Java and C is highly configurable I got some comments.  I have been looking at the different pull requests associated with this. In general I like what I seen regarding Java code but not so much what I seen on C code.  I don't know whether the JSS team has agreed on code style or not. Upstream Mozilla NSS has a sensible code guidelines at https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/NSS_Developer_Tutorial#nss_coding_style 
 my 2¢

File: src/main/java/org/mozilla/jss/ssl/PrintOutputStreamWriter.java [hunk 1]
```diff
@@ -13,28 +13,28 @@
...
-    extends java.io.OutputStreamWriter
+class PrintOutputStreamWriter
+	extends java.io.OutputStreamWriter
 {
 
-    public PrintOutputStreamWriter(OutputStream out)
-    {
-    	super(out);
-    }
+public PrintOutputStreamWriter(OutputStream out)
```
Comment (cipherboy) [inline src/main/java/org/mozilla/jss/ssl/PrintOutputStreamWriter.java:20]: I'd reasonably expect this to be indented, do others agree? I think this is a C++-ism more so than a Java-ism. Thoughts?

Comment (edewata) [reply src/main/java/org/mozilla/jss/ssl/PrintOutputStreamWriter.java:20]: Agree. I think we probably should stick with Eclipse for autoformatting Java code, and leave C/Python code unchanged for now.

Comment (cipherboy) [reply src/main/java/org/mozilla/jss/ssl/PrintOutputStreamWriter.java:20]: Did you see the issue with Eclipse in the Java code though? It completely murdered a nicely formatted enum declaration.

Comment (edewata) [reply src/main/java/org/mozilla/jss/ssl/PrintOutputStreamWriter.java:20]: Ah yes, that's really bad.

**Pull request closed without merging**